components/src/core/score-card/score-card.component.scss

```diff
@@ -76,8 +74,8 @@
     align-items: center;
     .pxb-score-card-badge-wrapper {
         z-index: 1;
-        margin-left: 16px;
-        margin-right: 16px;
+        margin-left: 1rem;
+        margin-right: 1rem;
```

Comment:
horizontal margin should be fixed value, 16px, to save some more space here

Reply:
Updated this,